pilot/cmd/pilot-agent/status/dialer_others.go

```diff
@@ -0,0 +1,41 @@
+//go:build !windows
+// +build !windows
+
+// Copyright Istio Authors
```

Comment:
Should the copyright remain the Kubernertes Authors? Not sure what the best practice is here.

Reply:
Yeah I think so, thanks for reminding